samples/apps/copilot-chat-app/webapi/CopilotChat/Controllers/DocumentImportController.cs

```diff
@@ -207,6 +239,27 @@ private async Task<string> ReadTxtFileAsync(IFormFile file)
         return await streamReader.ReadToEndAsync();
     }
 
+    /// <summary>
+    /// Read the content of a text file.
+    /// </summary>
+    /// <param name="file">An IFormFile object.</param>
+    /// <returns>A string of the content of the file.</returns>
+    private async Task<string> ReadImageFileAsync(IFormFile file)
```

Comment:
Since Tesseract is an OCR engine, I will rename this to `ReadTextFromImageFileAsync` or similar.

Reply:
I have renamed that method, thank you.